StreamChatSwiftUITests/Tests/ChatChannel/MessageComposerViewModel_Tests.swift

```diff
@@ -434,15 +434,15 @@ class MessageComposerViewModel_Tests: StreamChatTestCase {
         let viewModel = makeComposerViewModel()
         
         // When
-        for _ in 0..<10 {
+        for _ in 0..<30 {
```

Comment:
I fixed this in the other PR (set the limit back to 10 in the tests). This one and the other one below should be reverted.